client/src/app/site/motions/modules/motion-detail/components/motion-content/motion-content.component.ts

```diff
@@ -429,6 +435,20 @@ export class MotionContentComponent extends BaseComponent implements OnInit, OnD
                     contentPatch['text_' + paragraphNo] = amendmentParagraph;
                 }
             });
+            // If the motion has been shortened after the amendment has been created, we will show the paragraphs
+            // of the amendment as read-only
+            /**
+             * How to iterate over all amendment paragraphs in os4?
+             */
+            // for (
+            //     let paragraphNo = paragraphsToChoose.length;
+            //     paragraphNo < formMotion.amendment_paragraphs.length;
+            //     paragraphNo++
+            // ) {
+            //     if (formMotion.amendment_paragraphs[paragraphNo] !== null) {
+            //         contentPatch.broken_paragraphs.push(formMotion.amendment_paragraphs[paragraphNo]);
+            //     }
+            // }
```

Comment:
@GabrielInTheWorld relevant for checking, not obvious how OS4 motions encapsulate lists of motion amendments